bitcurves/bitcurve.go

```diff
@@ -59,25 +61,34 @@ func (BitCurve *BitCurve) IsOnCurve(x, y *big.Int) bool {
 	return x3.Cmp(y2) == 0
 }
 
```

Comment:
Checked that following function is okay.
`ModInverse` from the `big` package returns `nil` if there is no multiplicative inverse. Since `BitCurve.P` is a hardcoded prime, the only case for this is `Z=0`, therefore an equivalent way of doing this is to check if `zinv == nil`.

Reply:
Also added this case as a returned error.